release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -293,7 +311,72 @@ For the OVN-Kubernetes network plug-in, egress firewalls support audit logging u
+
+This feature automatically translates in-tree objects to their counterpart CSI representations and should be completely transparent to users. Translated objects are not stored on disk, and user data is not migrated.
+
+While storage class referencing to the in-tree storage plug-in will continue working, it is recommended that you switch the default storage class to the CSI storage class.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-migration.adoc[CSI Automatic Migration].
+
+[id="ocp-4-12-storage-gce-auto-migration-ga"]
+==== Automatic CSI migration for GCP PD auto migration is generally available
+Starting with {product-title} 4.8, automatic migration for in-tree volume plug-ins to their equivalent Container Storage Interface (CSI) drivers became available as a Technology Preview feature. Support for Google Compute Engine Persistent Disk (GCP PD) was provided in this feature in {product-title} 4.9, and {product-title} 4.12 now supports automatic migration for GCP PD as generally available. CSI migration for GCP PD is now enabled by default and requires no action by an administrator.
+
+This feature automatically translates in-tree objects to their counterpart CSI representations and should be completely transparent to users. Translated objects are not stored on disk, and user data is not migrated.
+
+While storage class referencing to the in-tree storage plug-in will continue working, it is recommended that you switch the default storage class to the CSI storage class.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-migration.adoc[CSI Automatic Migration].
+
+[id="ocp-4-12-storage-capacity-tracking"]
+==== Storage capacity tracking for pod scheduling is generally available
+This new feature exposes the currently available storage capacity using `CSIStorageCapacity` objects, and enhances scheduling of pods that use Container Storage Interface (CSI) volumes with late binding. Currently, the only {product-title} storage type that supports this features is OpenShift Data Foundation.
+
+For more information, see ???.
+
+[id="ocp-4-12-vsphere-topology-awareness"]
+==== VMware vSphere CSI topology awareness is generally available
+{product-title} provides the ability to deploy {product-title} for vSphere on different zones, which allows you to deploy over multiple data centers and clusters, thus helping to avoid a single-point-of-failure.
+
+For more information, see ???.
```

Comment:
I understand we need a link after https://github.com/openshift/openshift-docs/pull/52116 is merged, right?
Others look good to me.

Reply:
Correct. I will add the link after I've merged the top awareness PR.